### PR TITLE
add simplification as suggested by @yarden-livnat

### PR DESCRIPTION
**Pull request by bam241**
This should improve the efficiency of the timeseries methods

### CODE REVIEW

**Comment by bam241** [comment]
cc @yarden-livnat @FlanFlanagan

File: cymetric/timeseries.py [hunk 1]
```diff
@@ -224,15 +203,13 @@ def get_power(evaler, facilities=()):
...
 
     base_col = ['SimId', 'AgentId']
     added_col = base_col + ['Prototype']
-    power = tools.merge(power, base_col, agents, added_col)
+    df = tools.merge(df, base_col, agents, added_col)
 
-    group_end = ['Time']
-    group_start = group_end + ['Value']
-    df = power[group_start].groupby(group_end).sum()
+    df = df[['Time','Value']].groupby(['Time']).sum()
```
[the remark this answers is not in the document]
Comment (katyhuff) [reply cymetric/timeseries.py]:
```suggestion
    df = df[['Time', 'Value']].groupby(['Time']).sum()
```

File: cymetric/timeseries.py [hunk 2]
```diff
@@ -264,9 +241,7 @@ def get_deployment(evaler, facilities=()):
     # Adding a constant column to easily sum the amount of facilities build per
     # time step
     df = df.assign(Value=lambda x: 1)
-    group_end = ['EnterTime']
-    group_start = group_end + ['Value']
-    df = df[group_start].groupby(group_end).sum()
+    df = df[['EnterTime','Value']].groupby(['Time']).sum()
```
[the remark this answers is not in the document]
Comment (katyhuff) [reply cymetric/timeseries.py]:
```suggestion
    df = df[['EnterTime', 'Value']].groupby(['Time']).sum()
```

**Comment by bam241** [comment]
@katyhuff yes they leave in `test/test_filters.py` and `test/test_timeseries.py`

**Comment by katyhuff** [comment]
Looks great to me!

**Pull request merged**